_posts/2017-06-21-Devoxx-pl.md

```diff
@@ -105,10 +105,74 @@ To build up this discipline, automated verification can be seen as the software
 
 Extended blog coming later.
 
+## Feature Branches And Toggles In A Post-GitHub World ([Sam Newman](https://twitter.com/@samnewman))
+
+Sam told us about his experience at a project that was having trouble merging branches.
+The release branch for the next release was called R3, but for a large refactoring, branch R4 was created.
+Afterwards he described merging the branches as a car crash.
+They even needed to introduce a dedicated R3-R4 merge bug fix team.
+Then **Continuous Integration** was meant to prevent the merging problem.
```

Comment:
Would rephrase:
**Continuous Integration** had been set up in order to prevent the merging issues.